src/WebJobs.Script/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -50,6 +50,11 @@ public static bool IsAntaresInternalRequest(this HttpRequestMessage request)
             return !request.Headers.Contains(ScriptConstants.AntaresLogIdHeaderName);
         }
 
+        public static bool IsAuthDisabled(this HttpRequestMessage request)
```

Comment:
Yeah, good point, I thought about that too, the fact that this isn't strictly a property of the request. However, you can imagine that this might vary per request in the future, so having it be named as a request property makes it more general, so I think this is ok. I.e. the method returns a value indicating whether auth is disabled for this request. Currently this is a host level setting, but doesn't have to be.